src/model/immutable/__tests__/EditorState-test.js

```diff
@@ -299,6 +299,15 @@ describe('EditorState', () => {
       );
     });
 
+    it('must call decorator with correct argument types and order', () => {
+      var decorator = new Decorator();
+      getDecoratedEditorState(decorator);
+      decorator.getDecorations.mock.calls.forEach((call) => {
+        expect(call[0] instanceof ContentBlock).toBe(true);
```

Comment:
Is this sufficient? Was the best way I could think to test the API, but it feels a bit scant.